### ISSUE TITLE
A11y_MySQLextensionforAzureDataStudio_New deployment Azure Database for MySQL_Name Role Value:Learn more Microsoft azure terms of use and Microsoft Privacy Statement defined as button instead of link.

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio-mysql#187]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev3; #DesktopApp; #GH-MySQLextensionforAzureDataStudio-Win32-Mar23; #MySQL extension for Azure Data Studio; #Win32; #WCAG4.1.2; #Name Role Value; #Benchmark; #AILimited; 

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Environment and OS details:
Azure Data Studio Version: 1.42.0 (user setup)
Windows 11 Enterprise Version	22H2
Installed on	‎28-‎12-‎2022
OS build	22621.1413
Experience	Windows Feature Experience Pack 1000.22639.1000.0
### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till  new connection and press enter 
4. Enter a valid server name username and password and press connect
### Repro Steps:

1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment"
4. In the Deployment options, select "Azure Database For MySQL" card.
5. Tab till Microsoft azure terms of use and Microsoft Privacy Statement
6. Verify that Learn more Microsoft azure terms of use and Microsoft Privacy Statement  defined as link or not
### Actual Result:
Learn more Microsoft azure terms of use and Microsoft Privacy Statement defined as button instead of link.
### Notes:
Similar issue repro on Dashboard page also.
### Expected Result:
Learn more Microsoft azure terms of use and Microsoft Privacy Statement should be defined as a link.
### Attachment:
![A11y_MySQLextensionforAzureDataStudio_New deployment Azure Database for MySQL_Name Role Value](https://user-images.githubusercontent.com/100189407/228875988-2559ab1f-55a3-4dd3-8236-83a151d0396e.gif)

**Comment by nitishsharma25** [comment]
This issue is external to MySQL extension as it is a common UI of Azure Data Studio.

**Comment by VijayaBharathyA** [comment]
**Existing Bug Found :**
https://github.com/microsoft/azuredatastudio/issues/22096

**Comment by keerthiduvvuri** [comment]
GithubTags:#ResolvedByTT
